great_expectations/experimental/datasources/filesystem_data_asset.py

```diff
@@ -53,6 +153,77 @@ def build_batch_request(
 
         return super().build_batch_request(options)
 
+    def get_batch_list_from_batch_request(
```

Comment:
@NathanFarmer This code is proposed to be moved to `FilePathDataConnector` (it is general for all file-based assets).